packages/cloud_firestore/cloud_firestore_web/lib/src/load_bundle_task_web.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:async';
+
+import 'package:cloud_firestore_platform_interface/cloud_firestore_platform_interface.dart';
+import 'interop/firestore.dart';
+
+class LoadBundleTaskWeb extends LoadBundleTaskPlatform {
+  LoadBundleTaskWeb(this._task) : super() {
+    _controller = StreamController<LoadBundleTaskSnapshotPlatform>.broadcast(
```

Comment:
Same comments about the other similar class (.stream == .stream & potentially refactor to async*/map)